packages/teleterm/src/ui/services/usageEvent/usageEventService.ts

```diff
@@ -35,17 +40,20 @@ export class UsageEventService {
   ) {}
 
   captureLogin(uri: ClusterOrResourceUri): Promise<void> {
-    const clusterParams = this.getClusterProperties(uri);
-    if (!clusterParams) {
+    const clusterProperties = this.getClusterProperties(uri);
+    if (!clusterProperties) {
       return;
     }
-    return this.reportUsageEvent({
-      loginEvent: {
-        clusterProperties: clusterParams,
-        arch: this.runtimeSettings.arch,
-        os: this.runtimeSettings.platform,
-        osVersion: this.runtimeSettings.osVersion,
-        connectVersion: this.runtimeSettings.connectVersion,
+    const { arch, platform, osVersion, connectVersion, dev } =
+      this.runtimeSettings;
+    return this.reportUsageEvent(clusterProperties.authClusterId, {
+      userLogin: {
+        clusterName: clusterProperties.clusterName,
+        userName: clusterProperties.userName,
+        arch,
+        os: platform,
+        osVersion,
+        connectVersion: dev ? `${connectVersion}-dev` : connectVersion,
```

Comment:
Eh, there is one problem here, in dev mode `app.getVersion()` returns Electron version, not `package.json` version. It is caused by the way we run Electron. In `scripts` we have
```json
"start-electron": "electron build/app/dist/main/main.js"
```
so we just run the JS file. If we change it to 
```json
"start-electron": "electron ."
```
(so it can read `package.json`) then the app version is fine (`1.0.0-dev`). Unfortunately, it also changes the user data folder - from `Electron` to `Teleport Connect`. It means that if you have packaged app installed it will point to the same dir as a dev one. Technically, it seems to me that not using `Electron` dir is desired behaviour (otherwise you wouldn't be able to run two different apps in dev mode), but OTOH it is quite convenient to have dev/prod dirs separated (for example for testing). Or am I overthinking this and it is fine to have a single dir?

Reply:
How about executing `app.setVersion(process.env.npm_package_version)` in `MainProcess._init` if we see that we're using the dev version? I got the idea from here https://github.com/electron/electron/issues/7085#issuecomment-672268657. From what I see, [Electron should use the version set by `app.setVersion`](https://github.com/electron/electron/blob/31c2b5703a75a5ff2bf0e68f472d9002fe866bfb/shell/common/application_info_linux.cc#L58-L76).

> Technically, it seems to me that not using `Electron` dir is desired behaviour (otherwise you wouldn't be able to run two different apps in dev mode), but OTOH it is quite convenient to have dev/prod dirs separated (for example for testing). Or am I overthinking this and it is fine to have a single dir?

I also think that having two different dirs is super convenient, especially since I use the prod version of Connect to work on Connect.

Reply:
Unfortunately, `app.setVersion` is not a part of electron public API. I decided to just return `process.env.npm_package_version` from `getRuntimeSettings` in dev mode.